<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Euclid::PhzQtUI::MainWindow</class>
 <widget class="QMainWindow" name="Euclid::PhzQtUI::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>731</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="page_layout">
      <item>
       <widget class="QStackedWidget" name="mainStackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="btn_HomeToOption">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Working directory path definition, Auxiliary data management and Cosmological parameters definition.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Configuration </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_HomeToCatalog">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>Define the Mapping between the catalog file(s) columns and the filters</string>
               </property>
               <property name="text">
                <string>Catalog Setup</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_HomeToModel">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>Specify the parameter space for the creation of model grid.</string>
               </property>
               <property name="text">
                <string>Parameter Space</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_HomeToAnalysis">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Compute the model grid, calculate photometric correction, specify luminosity prior and run template fitting in order to produce photo-z catalog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Compute Redshifts </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_HomeToPP">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Computing PDF statistics, residuals between data and best fitted models, plots scatter, PIT and CRPS &lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Post Processing</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_exit">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Quit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Phosphoros</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Template Fitting Tool</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>For Photometric Redshift</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Euclid::PhzQtUI::ScalableImageLabel" name="image_label" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>400</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="Euclid::PhzQtUI::FormModelSet" name="widget_ModelSet" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="Euclid::PhzQtUI::FormSurveyMapping" name="widget_Catalog" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_5">
         <layout class="QVBoxLayout" name="verticalLayout_26">
          <item>
           <widget class="Euclid::PhzQtUI::FormAnalysis" name="widget_Analysis" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="Euclid::PhzQtUI::FormConfiguration" name="widget_configuration" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_6">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Euclid::PhzQtUI::FormPostProcessing" name="widget_postprocessing" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Euclid::PhzQtUI::FormSurveyMapping</class>
   <extends>QWidget</extends>
   <header>PhzQtUI/FormSurveyMapping.h</header>
  </customwidget>
  <customwidget>
   <class>Euclid::PhzQtUI::FormModelSet</class>
   <extends>QWidget</extends>
   <header>PhzQtUI/FormModelSet.h</header>
  </customwidget>
  <customwidget>
   <class>Euclid::PhzQtUI::FormAnalysis</class>
   <extends>QWidget</extends>
   <header>PhzQtUI/FormAnalysis.h</header>
  </customwidget>
  <customwidget>
   <class>Euclid::PhzQtUI::ScalableImageLabel</class>
   <extends>QWidget</extends>
   <header>PhzQtUI/ScalableImageLabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Euclid::PhzQtUI::FormConfiguration</class>
   <extends>QWidget</extends>
   <header>PhzQtUI/FormConfiguration.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Euclid::PhzQtUI::FormPostProcessing</class>
   <extends>QWidget</extends>
   <header location="global">PhzQtUI/FormPostProcessing.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PhUIResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>Euclid::PhzQtUI::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>679</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
